test/integration/krane_deploy_test.rb

```diff
@@ -1122,7 +1122,10 @@ def test_bad_container_on_stateful_sets_fails_with_rolling_update
   end
 
   def test_on_delete_stateful_sets_are_not_monitored
-    result = deploy_fixtures("hello-cloud", subset: ["stateful_set.yml"])
+    result = deploy_fixtures("hello-cloud", subset: ["stateful_set.yml"]) do |fixtures|
+      stateful_set = fixtures['stateful_set.yml']['StatefulSet'].first
+      stateful_set['spec']['updateStrategy'] = { 'type' => Krane::StatefulSet::ONDELETE }
```

Comment:
Is this because the default behavior changed?

Reply:
Yes, the default `updateStrategy`  for `StatefulSet API` for `v1beta1` is `OnDelete`, but in `v1` it is now `RollingUpdate`.

I added a comment to add more context in this section of the code.

For more information: 

- https://github.com/kubernetes/kubernetes/issues/56806#issuecomment-350197966
- https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#statefulsetupdatestrategy-v1beta1-apps
- https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#statefulsetspec-v1-apps